hasjob/views/listing.py

```diff
@@ -1017,6 +1017,28 @@ def close(domain, hashid, key):
     return render_template("close.html", post=post, form=form)
 
 
+@app.route('/<domain>/<hashid>/delete', methods=('GET', 'POST'), defaults={'key': None}, subdomain='<subdomain>')
+@app.route('/<domain>/<hashid>/delete', methods=('GET', 'POST'), defaults={'key': None})
```

Comment:
Drafts don't have a confirmed `domain`, so this URL should be `/delete/<hashid>`.